apps/web-mzima-client/src/app/settings/roles/role-item/role-item.component.ts

```diff
@@ -136,6 +134,7 @@ export class RoleItemComponent implements OnInit {
         },
       });
     } else {
+      roleBody.name = this.role.name; // Use role name from API - We don't want to change name property along with display_name on edit/update
       this.rolesService.updateRole(this.role.id, this.form.value).subscribe({
```

Comment:
The change is correct, but you arent submitting the roleBody in the updateRole call.

Reply:
You mean like this?
````
this.rolesService.updateRole(this.role.id, roleBody).subscribe({
````

Reply:
Ok, I think I now understand the problem with my code it's not just there 🤭